rules/windows/command_and_control_remote_file_copy_scripts.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/11/29"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies cscript.exe or wscript.exe being used to download an executable file from a remote destination.
```

Comment:
```suggestion
Identifies built-in Windows script interpreters (cscript.exe or wscript.exe) being used to download an executable file from a remote destination.
```

Reply:
What about other interpreters like cmd, powershell / pwsh? Would they be useful additions?

Reply:
different noise level, w|cscript are less verbose, powershell is captured by another PR.